src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/InputAbbreviationAsWordInNameType7.java

```diff
@@ -0,0 +1,28 @@
+/*
+AbbreviationAsWordInName
+allowedAbbreviations = ORDER, OBSERVATION, UNDERSCORE, TEST
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abbreviationaswordinname;
+
+public class InputAbbreviationAsWordInNameType7 {
+    void getTEST() {
+    } // OK
+
+    void getORDER_OBSERVATION() {} // ok
+
+    void getUNDERSCORE() {} // ok
+
+    void getTEST_OBSERVATION() {} // ok
+
+    void getTEST_UNDERSCORE() {} // ok
+
+    void getORDER() {} // ok
+
+    void getOBSERVATION() {} // ok
+
+    void getORDER_UNDERSCORE() {} // ok
```

Comment:
Please extend testing with cases where violation should be.

Reply:
Done

Reply:
Not done

Reply:
Ok ?